edx_exams/apps/core/api.py

```diff
@@ -214,3 +216,46 @@ def create_exam_attempt(exam_id, user_id):
     )
 
     return attempt.id
+
+
+def get_exam_by_content_id(course_id, content_id):
+    """
+    Retrieve an exam filtered by a course_id and content_id
+    """
+    try:
+        exam = Exam.objects.get(course_id=course_id, content_id=content_id)
+        return ExamSerializer(exam).data
```

Comment:
Navigating circular dependencies is a sure sign something is off. I think the root of that is that the core app should not be importing from the API. Personally, I don't think serializers belong in this file or the core app at all and we should be doing that in the view.

Reply:
The reason I was adding serializers to the api.py file was to avoid having the view handle any models directly.